src/core/server/integration_tests/saved_objects/serverless/migrations/smoke.test.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import {
+  request,
+  TestServerlessESUtils,
+  TestServerlessKibanaUtils,
+  createTestServerlessInstances,
+} from '@kbn/core-test-helpers-kbn-server';
+
+/**
+ * Until we merge https://github.com/elastic/kibana/pull/162673 this test should remain skipped.
+ */
+describe.skip('smoke', () => {
```

Comment:
I tried to get this working on CI, but it is currently timing out. It seems as though the instance might be to small to start 3 ES containers. We should also wait for QA to figure this out for FTRs.